src/nvme/ioctl.h

```diff
@@ -3855,4 +3855,16 @@ int nvme_zns_append(struct nvme_zns_append_args *args);
  */
 int nvme_dim_send(struct nvme_dim_args *args);
 
+/**
+ * nvme_debug - Set NVMe command debugging output
```

Comment:
kdoc is not happy because `nvme_debug` should be `nvme_set_debug`. The same problem with the kdoc section below

Reply:
Fixed the kdoc also. Thank you.